### ISSUE TITLE
Serializing to ids

### ISSUE THREAD

**Issue by benjaminledet** [ManueGE/Boxfit#5]
Hi, thanks for your amazing library.
In the features part you explain that which is really great : 

![capture d ecran 2018-07-09 a 13 23 28](https://user-images.githubusercontent.com/23617972/42447779-4c9877b0-837b-11e8-8523-69544a2fba29.png)

But can we do the reverse ? Send to the server only the ids of the array when we put an entity?

**Comment by ManueGE** [comment]
Hello!

Not possible at the moment, it shouldn't be difficult to add, but I'd need to find some time. 

In the meantime, you have another choice. You can create a `JSONObjectTransformer` like this:

```java
public class MyEntityTransformer implements JSONObjectTransformer {
    @Override
    public JSONObject transform(JSONObject object) {
        // return same object, you don't want your incoming JSON to be transformed 
        return object;
    }

    @Override
    public JSONObject inverseTransform(JSONObject object) {
        // modify the JSON object to replace your entities to their ids. Should be easy. 
        return object;
    }
}
```

Then in your entity, you need to add the `@BoxfitClass` annotation like this:

```java
@BoxfitClass(transformer = MyEntityTransformer.class)
```

Try this solution and when I find a gap I will update the library.

**Comment by ManueGE** [comment]
Awesome. Let's keep this issue open then. I will try to add it soon.

**Comment by ManueGE** [comment]
Should be fixed in `0.0.5`. 

Annotate the field with `@ToJsonAsId`.
